addons/point_of_sale/data/orders_demo.xml

```diff
@@ -1,190 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Used in https://github.com/odoo/odoo/blob/3ce510275636fff974781c1e2baa55b64f5b12bb/addons/point_of_sale/models/pos_config.py#L988